code/modules/jobs/job_types/assistant.dm

```diff
@@ -62,6 +63,10 @@ Assistant
 	else
 		uniform = GLOB.colored_assistant.jumpskirts[index]
 
+/datum/outfit/job/assistant/proc/give_maintenance_card(mob/libing/carbon/human/target)
```

Comment:
```suggestion
/datum/outfit/job/assistant/proc/give_maintenance_card(mob/living/carbon/human/target)
```

Reply:
I did see this and the failing linters but I was leaving home already so I can only fix it in a few hours.
Still, thank you for pointing it out.